test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionExecutorTests.cs

```diff
@@ -32,24 +32,22 @@ public class ControllerActionExecutorTests
 
         private delegate dynamic ReturnTaskAsDynamicValue(int i, string s);
 
-        [Fact]
-        public async Task AsyncAction_WithVoidReturnType()
+        [Theory,
+         InlineData(true),
+         InlineData(false)]
```

Comment:
just close each attribute instead of using a comma, there's nothing like this elsewhere in the codebase.